docs/stats.rst

```diff
@@ -70,6 +70,12 @@ e.g., the sites of the SNPs.
 Windowing
 *********
 
+By default, statistics
```

Comment:
I started writing this, and then decided to see how easy it would be to implement before getting into it.